puppet/modules/quickstack/manifests/pacemaker/redis.pp

```diff
@@ -0,0 +1,40 @@
+class quickstack::pacemaker::redis(
+  $bind_host             = '127.0.0.1',
+  $port                  = '6379',
+  $slaveof               = undef,
+) {
+
+  $redis_group = map_params("redis_group")
+
+  class {'::quickstack::firewall::redis':
+    ports => [$port],
+  }
+
+  class {'::quickstack::db::redis':
+    bind_host => $bind_host,
+    port      => $port,
+    slaveof   => $slaveof,
+  }
+
+  quickstack::pacemaker::resource::generic {'redis':
```

Comment:
Please alter this slightly to be consistent with the approach taken in PR 502, for example here: https://github.com/redhat-openstack/astapor/pull/502/files#diff-4d51d8210540212d092be4f5ca299483L136